docs/.vitepress/theme/CustomLayout.vue

```diff
@@ -0,0 +1,106 @@
+<script setup>
+import DefaultTheme from "vitepress/theme";
+
+const { Layout } = DefaultTheme;
+</script>
+
+<template>
+  <Layout>
+    <template #layout-top
+      ><a class="banner" href="https://zuplo.com/" target="_blank">
```

Comment:
Can you change this to `https://zuplo.link/openapi-ts-web` it lets us track better. On github, please use `https://zuplo.link/openapi-ts-gh` so we can differentiate the traffic from your website vs github